distribution/src/assembly/assembly-descriptor-slim.xml

```diff
@@ -48,7 +48,7 @@
         <file>
             <source>${project.build.directory}/classes/META-INF/THIRD-PARTY.txt</source>
             <destName>THIRD-PARTY.txt</destName>
-            <outputDirectory>licenses</outputDirectory>
+            <outputDirectory>/</outputDirectory>
```

Comment:
Is there a requirement for the THIRD-PARTY.txt to be in root? It's polluting the root folder with yet another useless file.

Reply:
reverted